### ISSUE TITLE
loss值震荡不下行

### ISSUE THREAD

**Issue by dhh123** [aolingwen/fuck_verifycode#1]
>> 已训练27110个批次: loss = 3.53 (2.757 sec), 该批正确数量 = 0
>> 已训练27120个批次: loss = 3.53 (2.809 sec), 该批正确数量 = 0
>> 已训练27130个批次: loss = 3.53 (2.825 sec), 该批正确数量 = 0
>> 已训练27140个批次: loss = 3.53 (2.845 sec), 该批正确数量 = 0
>> 已训练27150个批次: loss = 3.53 (2.714 sec), 该批正确数量 = 0
>> 已训练27160个批次: loss = 3.53 (2.745 sec), 该批正确数量 = 0
>> 已训练27170个批次: loss = 3.53 (2.788 sec), 该批正确数量 = 0

**Comment by dhh123** [comment]
nZ
